src/components/KeywordFinder/HomeKeywordFinderSection.tsx

```diff
@@ -489,19 +489,13 @@ const HomeKeywordFinderSection: React.FC<HomeKeywordFinderSectionProps> = props
             -webkit-overflow-scrolling: touch;
           `}>
           {genreKeywords.map((keyword, index) => {
-            if (searchParam.has('set_id')) {
-              searchParam.delete('set_id');
-            }
-            if (searchParam.has('tag_ids[]')) {
-              searchParam.delete('tag_ids[]');
-            }
-
-            searchParam.append('set_id', keyword.set_id.toString());
-            searchParam.append('tag_ids[]', keyword.tag_id.toString());
+            const keywordSetAndTagSearchParam = new URLSearchParams();
```

Comment:
이렇게 기존에 있던 `URLSearchParams`를 복사할 수 있더라고요.

```suggestion
            const keywordSetAndTagSearchParam = new URLSearchParams(genreSearchParam);
```